x-pack/metricbeat/module/gcp/gke/_meta/docs.asciidoc

```diff
@@ -0,0 +1,31 @@
+`gke` metricset is designed for collecting metrics from Google Kubernetes
+Engine. GKE is the industry's first fully managed Kubernetes service that
```

Comment:
_GKE is the industry's_... I'm being picky here but I'd rather talk about Beat's product than talk about Google's product. What is GKE and why is (or not) awesome should be explained (and sold) in GCP GKE's homepage. All in all, this is the "homepage" of Metricbeat GKE metricset for GCP module.

Reply:
Agreed. I added it because I found the explanation for the monitored service in other metricsets and I copy/pasted from the GKE official page. I'll replace it with a link to the service home page.